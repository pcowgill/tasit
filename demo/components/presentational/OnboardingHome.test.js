import React from "react";
import { shallow } from "enzyme";
import OnboardingHome from "./OnboardingHome";

describe("OnboardingHome", () => {
  it("renders the component", async () => {
    expect(shallow(<OnboardingHome onPress={() => {}} />)).toMatchSnapshot();
  });
});
