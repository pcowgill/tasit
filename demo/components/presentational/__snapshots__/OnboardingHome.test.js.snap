// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnboardingHome renders the component 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Component
    source={1}
  />
  <LargeText>
    Let's get you set up with a secure way to store this land!
  </LargeText>
  <Component
    style={
      Object {
        "flexDirection": "row",
        "marginTop": 66.7,
      }
    }
  >
    <Button
      onPress={[Function]}
      title="Get started"
    />
  </Component>
</Component>
`;
